docs/releases/2.2.txt

```diff
@@ -173,6 +173,15 @@ Management Commands
 * The new :option:`--force-color` option forces colorization of the command
   output.
 
+* The :option:`inspectdb --include-views` option now also allows creating
+  models for materialized views in PostgreSQL.
+
+* The :djadmin:`inspectdb` command now supports creating models for foreign
+  tables in PostgreSQL.
+
+* The new :option:`inspectdb --include-partitions` option allows creating
+  models for partition tables.
```

Comment:
PostgreSQL?

Reply:
As mentioned above, this should not limited to PostgreSQL. Will extend to mention where the backend supports partitioning.